glean-core/csharp/csharp/Dispatchers.cs

```diff
@@ -0,0 +1,21 @@
+﻿// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
```

Comment:
You don't need using `System.Text`  and `System.Collections.Generic`

Reply:
Good point. Finalizing this will be part of [a separate bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1643011).